modules/bmc/bmc.rb

```diff
@@ -4,19 +4,19 @@ module Proxy
   module BMC
     # Just a bunch of stubs
     def installed_providers
-      Proxy::BMC::IPMI.providers_installed + ['shell']
+      Proxy::BMC::IPMI.providers_installed + ['redfish', 'shell']
     end
 
     def installed_ipmi_providers
       Proxy::BMC::IPMI.providers_installed
     end
 
     def providers
-      Proxy::BMC::IPMI.providers + ['shell']
+      Proxy::BMC::IPMI.providers + ['redfish', 'shell']
     end
 
     def installed?(provider)
-      if provider == 'shell'
+      if provider == 'redfish' or provider == 'shell'
```

Comment:
Ruby typically uses `||` rather than `or`
```suggestion
      if provider == 'redfish' || provider == 'shell'
```

Reply:
it uses both, and the precedence is slightly different, but I don't mind changing it. I'll include it in the next update.

Reply:
changed.